lib/helpers/HttpStatusCode.js

```diff
@@ -0,0 +1,67 @@
+var HttpStatusCode;
```

Comment:
Yes, the enum should be packaged and generated programmatically if it saves a few bytes, especially if it contains duplicate string literals, as it will be used in browsers too. Also, there is no named export in ./index.js. to support the ESM entry point.

Reply:
@DigitalBrainJS sorry, I'm not sure if you are suggesting to leave if or change it :see_no_evil:

> Also, there is no named export in ./index.js. to support the ESM entry point.

Thanks for catching this, added now. :+1:

Reply:
ah, thanks—updated now :heavy_check_mark: